ui/src/components/ConfigModalContent/index.tsx

```diff
@@ -151,6 +152,122 @@ export const ConfigModalContent: React.FC = () => (
       </ul>
       <p>Each customers key assigned to the other attributes must be unique.</p>
 
+      <Heading4>SEP-24</Heading4>
+      <p>
+        SEP-24 configuration objects are used for specifying the account
+        credentials and transaction parameters that should be used when making
+        GET /transaction requests for fetching pending and completed
+        transactions.
+      </p>
```

Comment:
Can we note that the config file is optional if the anchor doesn't want to set up transactions in these statuses before running tests?

Reply:
Done